src/reducers/userAccess.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers } from 'redux';
+import { PROMPT_LOGIN, SET_SERVER } from '../actions/userAccess';
+import { createActionReducer, createFlagReducer } from '../utils/genericReducer'
+
+export const userAccessReducer = combineReducers({
+	promptLogin: createFlagReducer(PROMPT_LOGIN),
+	server: createFlagReducer(SET_SERVER)
+});
+
+// set up for middleware operations 
+// helpful for making async and api calls
+export const userAccessMiddleware = [];
```

Comment:
Why keep the redux stuff in the library?

Reply:
My think was that this would be handy in the future and can be easily extended if need be, but you are right I would remove in the next push.